packages/core/src/dynamoDb/utils/editItem.ts

```diff
@@ -0,0 +1,158 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as fs from 'fs'
+import * as os from 'os'
+import * as path from 'path'
+import * as vscode from 'vscode'
+import { RowData, TableSchema, getAttributeValue } from './dynamodb'
+import { DynamoDbClient } from '../../shared/clients/dynamoDbClient'
+import { AttributeUpdates, GetItemInput, GetItemOutput, UpdateItemInput } from 'aws-sdk/clients/dynamodb'
+import { telemetry } from '../../shared/telemetry'
+
+export async function editItem(
+    selectedRow: RowData,
+    { tableName, regionCode, tableSchema }: { tableName: string; regionCode: string; tableSchema: TableSchema }
+) {
+    const currentContent = JSON.stringify(selectedRow, undefined, 4)
+    let filename = `${selectedRow[tableSchema.partitionKey.name]}`
+    if (tableSchema.sortKey) {
+        filename += `-${selectedRow[tableSchema.sortKey.name]}`
+    }
+    const tempFilePath = path.join(os.tmpdir(), `${filename}.json`)
```

Comment:
If we have time I wonder if we should re-visit if we can figure out a virtual file